src/AssemblerHttpServer-Alignments.cpp

```diff
@@ -1182,6 +1183,255 @@ void Assembler::computeAllAlignments(
 }
 
 
+void Assembler::sampleReads(vector<OrientedReadId>& sample, uint64_t n){
+    sample.clear();
+
+    while (sample.size() < n) {
+        // Randomly select a read in the read set
+        uint16_t readId = uint16_t(rand() % reads.size());
+
+        // Randomly select an orientation
+        uint16_t strand = uint16_t(rand() % 2);
+
+        const OrientedReadId r(readId, strand);
+
+        sample.push_back(r);
+    }
+}
+
+
+void Assembler::sampleReads(vector<OrientedReadId>& sample, uint64_t n, uint64_t minLength, uint64_t maxLength){
+    sample.clear();
+
+    while (sample.size() < n) {
+        // Randomly select a read in the read set
+        uint16_t readId = uint16_t(rand() % reads.size());
```

Comment:
Same here.

Reply:
Please use `ReadId`, currently defined as `uint32_t`. This is mostly for better readability, although the theoretical possibility that this will change one day in the distant future exists. Also I prefer to define as `const` local variables that are filled in on assignment and never changed.